### DISCUSSION TITLE
flutter_rust_bridge_codegen initialization inside a devContainer

### DISCUSSION THREAD

**Discussion by TibiToz** [fzyzcjy/flutter_rust_bridge#1896] [General]
I created a devcontainer for my flutter_rust_bridge app everything works fine except that to be able to `flutter run` my app, I need to do ` flutter_rust_bridge_codegen create tmp_app && rm -rf tmp_app` after every container reset and I'm not sure it will work correctly as it seems to depends on the path of the created project,.

I understand that `flutter_rust_bridge_codegen create` is doing some liking with flutter and cargo. Is it possible to reproduce the behaviour of this command but without creating a new project. But instead do the linking with an already created flutter project. That way I could integrate it to my Dockerfile or at the start of the container.

Thank for your help.

T.

**Comment by fzyzcjy** [comment]
Hi, `create` is doing nothing but copy-paste some files. Therefore, I suggest you just `create` once, and work on that folder later.

**Reply by TibiToz** [reply]
Are the copied files somehow link to the directory used by `create` ? Because if it is the case, it's a problem for me because I want to create a generic developmment  enviroment for flutter_rust_bridge.

**Reply by fzyzcjy** [reply]
IIRC there is no special links.

My suggestion is that, 

1. your devcontainer contains things like `flutter` binary, `cargo` binary, etc
2. you have a repo (say, https://github.com/TibiToz/your-app), and that repo contains some rust and dart file which is either your source code or flutter_rust_bridge's generated code. Just treat frb's code like your code (or gitignore them - but that's another story)

**Comment by TibiToz** [comment]
I had to run
`flutter pub get `
in
`my_app/rust_builder/cargokit/build_tool`
and
`flutter pub add rust_lib_myapp --path=rust_builder`
at the start of the container and everything works fine.

Thank you.

**Reply by fzyzcjy** [reply]
`flutter pub add rust_lib_myapp --path=rust_builder` IMHO this is not needed, since this is indeed just a script to change your `pubspec.yaml`.

**Reply by TibiToz** [reply]
You are right, it is not needed.
